scripts/casl_chain.py

```diff
@@ -187,6 +189,7 @@
     'Sb127': (False, 3, 2, None),
     'Te127': (False, 3, -1, None),
     'Te127_m1': (False, 3, -1, None),
+    'Te129': (False, 3, 2, None),
```

Comment:
The '2' indicates that the cumulative yield should be used. Since we have Te129_m1 in the chain which decays to Te129, wouldn't this be double counting that part of the yield?

Reply:
Yes. Independent yields are now used